.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tosca-yaml-parser"]
+	path = planner/core/tosca-yaml-parser
+	url = git@github.com:pqnet/tosca-yaml-parser.git
```

Comment:
This is breaking the checkout of this branch. Do we need to use gitmodules? What about including that project as a Maven dependency?